mlrun/config.py

```diff
@@ -325,6 +325,10 @@
         "requests": {"cpu": None, "memory": None, "gpu": None},
         "limits": {"cpu": None, "memory": None, "gpu": None},
     },
+    # default preemptible node selector to be applied when running on spot nodes - json string base64 encoded format
+    "default_preemptible_function_node_selector": "e30=",
+    # default preemptible tolerations to be applied when running on spot nodes - json string base64 encoded format
+    "default_preemptible_function_tolerations": "e30=",
```

Comment:
Now that I'm reconsidering the naming (should have done that on the Provazio PR), there's no default envolved here, you're telling MLRun what node selector and toleration are needed to be assigned to the function to make it run on a preemptible node, therefore I think we should go with similar approach to what was done in Nuclio (in general we should do that, it will make our life as developers of the two repositories easier):
* Add a `preemptible_nodes` block
* Add under it to values, `node_selector` and `tolerations`